js/init.js

```diff
@@ -0,0 +1,81 @@
+import dayjs from 'https://esm.archive.org/dayjs'
+import yml from 'https://esm.archive.org/js-yaml'
+import { isValidCustomElement, registerCustomElement } from './utils-wc.js'
+import {
+  getFromContext_DateConversion,
+  isContextRequest_DateConveresion,
+  isValidContextResponse_DateConversion,
+} from './context-date-conversion.js'
+import {
+  getFromContext_TransformMakup,
+  isContextRequest_TransformMakup,
+  isValidContextResponse_TransformMakup,
+  markdownReplaceYouTubeShortCodes,
+} from './context-markup.js'
+import { markupParser } from './defaults.js'
+
+/**
+ * Our own components, but not loading them just yet.
+ */
+const OUR_COMPONENTS = [
+  ['bt-time', './bt-date-time.js'],
+  ['bt-un-markup', './bt-un-markup.js'],
+]
```

Comment:
Provide default implementation of components to be registered (do not register them right away, allow people using us the ability to give us their implementation.